WordPressComKit/RequestRouter.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Alamofire
+
+public enum RequestRouter: URLRequestConvertible {
+    static let baseURLString = "https://public-api.wordpress.com/rest/v1.1/"
```

Comment:
The thing is, as far as I know, there's no such thing as "a version of the API" other than `v1`. Each endpoint is versioned separately: some endpoints have a 1.2 version (or even an undocumented 1.3), but those which don't will return a 404 if called with a version too high.

I don't like how we handle this in the app either, but the good thing is that here it's an internal implementation detail. A consumer of WordPressComKit would just call `MeService.fetchMe` and never care which version of the API it's using.